spec.v1.yml

```diff
@@ -2,9 +2,29 @@ swagger: '2.0'
 info:
   version: '1.0'
   title: Sunnah.com API
+  description: |-
+    Sunnah.com API offers access to verified data of collections of hadith of the Prophet (SAWS). Here you will find a list of the available endpoints in our REST API.
+
+    You will need an API key to access this data, which you may request from https://sunnah.com/contact.
```

Comment:
Can we instead have them open an issue on GitHub with some basic required information that we specify in the README? (name, email, purpose, rate requested)

Reply:
Added issue template for this